manifest.md

```diff
@@ -23,6 +23,7 @@ A client will distinguish a manifest list from an image manifest based on the Co
 
   This REQUIRED property contains the MIME type of the manifest list.
   For this version of the specification, this MUST be set to `application/vnd.oci.image.manifest.list.v1+json`.
+  For the MIME type that this is compatible with see the [matrix](media-types.md#compatibility-matrix).
```

Comment:
This doesn't have to be singular.  And most of the rest of the spec uses the generic “media type” instead of the email-specific “MIME type”.  So I'd suggest “MIME type” → “media types”.